lua/tinted-live-reload.lua

```diff
@@ -0,0 +1,44 @@
+local M = {}
+
+local function check_prerequisites()
+  local ok = pcall(require, "fwatch")
+  if not ok then
+    vim.notify("Required plugin rktjmp/fwatch.nvim is missing.", vim.log.levels.ERROR)
+    return false
+  end
+  return true
+end
+
+local function schedule_trigger(callback)
+  vim.schedule(callback)
+end
+
+local function start_watcher(callback)
+  if vim.g.tinted_live_reload_registered then
+    return
+  end
+  vim.g.tinted_live_reload_registered = true
+
+  local fwatch = require("fwatch")
+  local file_path = "~/.local/share/tinted-theming/tinty/current_scheme"
```

Comment:
```suggestion
  local file_path = vim.fn.system({ "tinty", "config", "--data-dir-path" }) .. "current_scheme"
```

Reply:
Not necessary now, but I think we should add a path option directly to Tinty so we don't rely on a specific path or filename in case it changes. So like 

```lua
local file_path = vim.fn.system({ "tinty", "config", "--current-scheme-path" })
```